tools/figma-inspector/src/main.tsx

```diff
@@ -146,7 +146,7 @@ export const App = () => {
                         style={{
                             color: exportsAreCurrent
                                 ? "var(--figma-color-text-disabled)"
-                                : "var(--figma-color-text-primary",
+                                : "",
```

Comment:
Seems odd to use an empty string for the color. Is that correct? Shouldn't the entire style attribute be omitted instead?

(Just looking at it from a html perspective, nevermind if react handles this)